bin/docker/supervisor.conf

```diff
@@ -10,3 +10,12 @@ redirect_stderr=true
 stdout_logfile=logs/docker.log
 stdout_logfile_maxbytes=1MB
 stopsignal=KILL
+
+[program:celeryd]
+command=python manage.py celeryd --broker=redis://redis:6379/0
```

Comment:
I thought so, but every combination i tried failed and it tried to connect to Rabbit, if someone can figure it out and it works, awesome.